### ISSUE TITLE
unknown probs for negative logits

### ISSUE THREAD

**Issue by elbaro** [abhijitbendale/OSDN#8]
```py
wscore = category_weibull[2][channel].w_score(channel_distance)
modified_fc8_score = channel_scores[categoryid] * ( 1 - wscore*ranked_alpha[categoryid] )
openmax_fc8_channel += [modified_fc8_score]
openmax_fc8_unknown += [channel_scores[categoryid] - modified_fc8_score ]
```
Can you explain how this works with negative fc8 scores?
As I understand, if v_i(x) is negative then even with w_i=1, v_0(x)=0, so v_i < v_0. 
In fact a fc8 layer may contain negative logits, and it leads to a high unknown-unknown probability in my dataset.

```py
features = loadmat('data/train_features/n01440764/n01440764_9981.JPEG.mat')
ch = 0
features['fc8'][ch]>0

array([ True,  True,  True,  True,  True,  True,  True,  True,  True,
       False,  True,  True,  True,  True,  True,  True,  True,  True,
       False,  True,  True,  True,  True,  True,  True,  True,  True,
        True,  True,  True,  True,  True,  True,  True,  True,  True,
        True,  True,  True,  True,  True,  True, False,  True,  True,
        True,  True,  True,  True,  True,  True,  True,  True,  True,
        True,  True,  True,  True,  True,  True,  True,  True,  True,
        True,  True,  True,  True,  True,  True, False, False,  True,
       False, False, False, False,  True, False, False,  True,  True,
       False,  True,  True,  True,  True,  True,  True,  True,  True,
...
```
